pyproject.toml

```diff
@@ -125,3 +124,79 @@ known-first-party = ["PIL"]
 [tool.pytest.ini_options]
 addopts = "-ra --color=yes"
 testpaths = ["Tests"]
+
+[tool.mypy]
+python_version = "3.8"
+pretty = true
+disallow_any_generics = true
+enable_error_code = "ignore-without-code"
+extra_checks = true
+follow_imports = "silent"
+warn_redundant_casts = true
+warn_unreachable = true
+warn_unused_configs = true
```

Comment:
https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-warn-unused-configs
> This requires turning off incremental mode using [--no-incremental](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-incremental)

Reply:
Let's remove this one then, the cache is more important.